website/docs/foundations/icons/index.js

```diff
@@ -10,6 +11,9 @@ const DEBOUNCE_MS = 250;
 export default class Index extends Component {
   @service router;
 
+  @tracked selectedIconSize =
```

Comment:
curious if you found it necessary to move this to be tracked here? I didn't do the same for `searchQuery` which may or may not be a problem actually.

Reply:
yes, otherwise it didn't work in updating the "selected" value in the `<select>` element (but if you want I can try to remove it and see what happens